### PR TITLE
*: expand the experimental_ui_max_stdouterr_bytes

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -23,7 +23,7 @@ test --test_output=errors --test_summary=detailed
 test:ci --color=yes
 test:ci --verbose_failures --test_verbose_timeout_warnings
 test:ci --test_env=GO_TEST_WRAP_TESTV=1
-test:ci --experimental_ui_max_stdouterr_bytes=104857600
+test:ci --experimental_ui_max_stdouterr_bytes=268435456
```
Comment (Defined2014) [inline .bazelrc:26]: Does it a magic number? And what does this control?

Comment (hawkingrei) [reply .bazelrc:26]: The maximum size of the stdout / stderr files that will be printed to the console. avoid collecting excessively large log files in CI. But it is too small for developers to check the log.

**Review by Defined2014 — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]

**Pull request merged**